misc/vtr-libs/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
this looks very similar to https://github.com/hdl/conda-eda/blob/master/pnr/vtr-gui/build.sh, I noticed https://github.com/hdl/conda-eda/tree/master/pnr/vtr does a symlink to the vtr-gui package, maybe something similar could be done here?